fe/fe-core/src/main/java/com/starrocks/catalog/TableProperty.java

```diff
@@ -127,6 +133,14 @@ public class TableProperty implements Writable, GsonPostProcessable {
     @SerializedName(value = "storageInfo")
     private StorageInfo storageInfo;
 
+    // partitionId -> binlogAvailabeVersion
+    private Map<Long, Long> binlogAvailabeVersions = new HashMap<>();
```

Comment:
We can remove `Partition#binlogAvailableVersion` ?

Reply:
At present, it is really unnecessary. The reason for not deleting it is that considering the need that it may be possible to obtain this attribute directly from the partition in the future.